modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java

```diff
@@ -166,6 +172,23 @@ private String _getSku(CPDefinition cpDefinition, Locale locale) {
 		);
 	}
 
+	private Map<String, String> _getUrlTitleMap(
+		Map<Locale, String> urlTitleMap) {
+
+		Set<Map.Entry<Locale, String>> entries = urlTitleMap.entrySet();
+
+		Stream<Map.Entry<Locale, String>> stream = entries.stream();
+
+		return stream.collect(
+			Collectors.toMap(
+				e -> {
+					Locale key = e.getKey();
+
```

Comment:
We should name this entry.
```suggestion
				entry -> {
					Locale key = entry.getKey();

```